Documentation for v11.0.0 (#610)
PostgREST/postgrest-docs: docs/api.rst

ilike         :code:`ILIKE`             ILIKE operator (to avoid `URL encoding <https://en.wikipedia.org/wiki/Percent-encoding>`_ you can use ``*`` as an alias of the percent sign ``%`` for the pattern)
match         :code:`~`                 ~ operator, see :ref:`pattern_matching`
imatch        :code:`~*`                ~* operator, see :ref:`pattern_matching`
all           :code:`ALL`               comparison matches all the values in the list,
                                        used as a modifier of :code:`eq,like,ilike,gt,gte,lt,lte,match,imatch`,
                                        e.g. :code:`?name=like(all).{J*,*e}`
any           :code:`ANY`               comparison matches any value in the list,
                                        used as a modifier of :code:`eq,like,ilike,gt,gte,lt,lte,match,imatch`,

>                                         e.g. :code:`?name=like(any).{J*,*e}`

Reviewer: This should be at the end of the table, next to the `or` and `and` operators.

It could also have a mention in the "Logical Operators" section. Some advantages:

- `any` values don't need double quoting unlike `in` values
- They really make more sense by using the `like/ilike` operators. `all` variant can be used as an example in combination with `like`.
- They're more of a shortcut. Check https://github.com/PostgREST/postgrest/issues/1569

    }
  ]

> If you just want to filter the films by actors but don't want to include them in the response, you must leave the embedded columns empty.

Reviewer: ```suggestion
If you want to filter the films by actors but don't want to include them in the response, you must leave the embedded columns empty.
```

.. tabs::

  .. code-tab:: http

    GET /films?select=title,actors!inner()&actors.first_name=eq.Jehanne HTTP/1.1

  .. code-tab:: bash Curl

    curl "http://localhost:3000/films?select=title,actors!inner()&actors.first_name=eq.Jehanne"

.. code-block:: json

  [
    {
      "title": "The Haunted Castle",
    }
  ]

> The top level resource can also be sorted by the embedded resource columns in :ref:`Many-to-One <many-to-one>` and :ref:`One-to-One <one-to-one>` relationships. For example, sorting the films by the director's last name in descending order:

Reviewer: ```suggestion
You can also sort the top-level resource by the embedded resource columns in :ref:Many-to-One <many-to-one> and :ref:One-to-One <one-to-one> relationships. For example, you could arrange the films in descending order using the director's last name.
```

.. tabs::

  .. code-tab:: http

    GET /films?select=title,directors(last_name)&order=directors(last_name).desc HTTP/1.1

  .. code-tab:: bash Curl

    curl "http://localhost:3000/films?select=title,directors(last_name)&order=directors(last_name).desc"

> The ``is.null`` filter can be used in embedded resources to perform an anti-join. For instance, to get all the films that do not have any nominations:

Reviewer: This link can be used on the release notes:

```suggestion
.. _null_embed:

Null filtering on embedded resources
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The ``is.null`` filter can be used in embedded resources to perform an anti-join. For instance, to get all the films that do not have any nominations:
```

    }
  ]

The top level resource can also be sorted by the embedded resource columns in :ref:`Many-to-One <many-to-one>` and :ref:`One-to-One <one-to-one>` relationships. For example, sorting the films by the director's last name in descending order:

.. tabs::

  .. code-tab:: http

    GET /films?select=title,directors(last_name)&order=directors(last_name).desc HTTP/1.1

  .. code-tab:: bash Curl

    curl "http://localhost:3000/films?select=title,directors(last_name)&order=directors(last_name).desc"

> 

Reviewer: This feature gets a bit lost with the current structure.

I think it should be below `Embedding with Top-level Filtering` and under the `"Top-Level Ordering"` subheading.

This way the feature can also be linked in the release notes.

Reviewer: "related orders" as the name for the feature will make more sense when we add "related filters".
